### ISSUE TITLE
Issue put request

### ISSUE THREAD

**Issue by Tona-Mangobyte** [chantouchsek/vue-axios-http#316]
@Chantouch @chantouchsek @sctouch 
I need put request without id
put<T>(payload: any):<T>

**Comment by chantouchsek** [comment]
you can extend it.

```js
putWithoutId<T>(payload: any) {
  return this.submit<T>('put', '/', payload)
}
```

**Comment by Tona-Mangobyte** [comment]
I see but I want use template method

![image](https://user-images.githubusercontent.com/55422610/162554927-ec5bfbbc-c31f-4374-824a-6d8509d70d22.png)

**Comment by Tona-Mangobyte** [comment]
and should be option clear parameter
removeParameters(['page', 'limit', 'sort', 'direction'])
in methods post put & delete

**Comment by chantouchsek** [comment]
> 

What do you mean, template method?

**Comment by Tona-Mangobyte** [comment]
template method is method build in.
Normally method build in. I call template method

**Comment by chantouchsek** [comment]
I see. @Tona-Mangobyte I see. 😄😁

**Comment by chantouchsek** [comment]
You can contribute, and I will check your PR 😁

**Comment by chantouchsek** [comment]
@Tona-Mangobyte check the PR, is that ok?

**Comment by Tona-Mangobyte** [comment]
@chantouchsek thanks

**Comment by chantouchsek** [comment]
Released: `v1.3.0`
